www/js/views/services.js

```diff
@@ -10,6 +10,7 @@
 
 var $ = require('jquery');
 
+var _ = require('underscore');
 var Backbone = require('backbone');
 var adminui = require('../adminui');
 var Services = require('../models/services');
```

Comment:
var Model = require('../models/model');  and replace below. also the same for collection